ilastik/workflows/carving/preprocessingSerializer.py

```diff
@@ -37,25 +37,30 @@ def _serializeToHdf5(self, topGroup, hdf5File, projectFilePath):
         preproc = topGroup
 
         for opPre in self._o.innerOperators:
-            mst = opPre._prepData[0]
+            mst = opPre.cachedResult[0]
```

Comment:
It would be nice to extract all cached values in a dataclass and pass it around instead of individual values. It's also unclear why `cachedResult` is a 1-element list and not just an MST value itself... OTOH, I'm not sure we should do these changes right now.